nnpdf_data/nnpdf_data/commondata/CMS_WCHARM_13TEV/metadata.yaml

```diff
@@ -1,65 +1,61 @@
 setname: CMS_WCHARM_13TEV
-version: 1
-version_comment: Port of old commondata
+
 nnpdf_metadata:
   nnpdf31_process: DY CC
   experiment: CMS
+
 arXiv:
   url: https://arxiv.org/abs/1811.10021
   journal: Eur. Phys. J. C 79 (2019) 269
 iNSPIRE:
-  url: ''
+  url: https://inspirehep.net/literature/1705068
 hepdata:
-  url: ''
-  version: -1
+  url: https://www.hepdata.net/record/ins1705068
+  version: 1
+
+version: 2
+version_comment: Implementation in the new format
+
 implemented_observables:
 - observable_name: WPWM-TOT-UNNORM
   observable:
     description: Jet Rapidity Distribution
     label: CMS $W+c$ total 13 TeV
-    units: ''
-  process_type: EWJ_RAP
-  tables: []
-  npoints: []
+    units: '[fb]'
+  process_type: DY_W_ETA
+  tables: ['WplusCharm_AbsEtaMu_Table']
   ndata: 5
   plotting:
-    kinematics_override: ewj_rap_sqrt_scale
+    kinematics_override: identity
     dataset_label: CMS $W+c$ total 13 TeV
     y_label: $d\sigma(W+c)/d|\eta^\mu|$ (fb)
     x_label: $|\eta^\mu|$
-    plot_x: k1
-  kinematic_coverage:
-  - k1
-  - k2
-  - k3
+    plot_x: abs_eta
+  kinematic_coverage: [abs_eta, m_W2]
```

Comment:
Why do the legacy kinematics have three variables and the new implementation just two? And does the legacy dataset not use this metadatafile and thus cause issues if the kinematics don't match?

From the value it seems the removed variable was just supposed to indicate the 13TeV beam energy, which I don't think would be used anywhere. But this question is more about how the code deals with it.

Reply:
The third variable in the old implementation was the beam energy. However, this is optional in the new common data parser as the $\sqrt{s}$ is automatically read from the name of the dataset. In other words, using sqrts here would be redundant.

The legacy dataset does use this metadata file. However, since the kinematics is the same between the two versions, the legacy file can be removed (and I will).